web/web/static/css/dtanm.css

```diff
@@ -2,6 +2,19 @@ body {
   overflow-x: hidden;
 }
 
+/* Header styling */
+.page-header {
+  position: sticky;
+  top: 0;
+  z-index: 1020;
+  background-color: var(--bs-body-bg);
+  transition: background-color 0.3s ease;
```

Comment:
This line makes the transition look a bit strange, IMO: 
[Screencast_20250521_233456.webm](https://github.com/user-attachments/assets/06084e54-d2ff-470d-85f7-c32867d2256d)